spid_cie_oidc/onboarding/tests/test_04_authn_response.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from django.test import TestCase
+from spid_cie_oidc.onboarding.validators.authn_response import (
+    AuthenticationErrorResponse, AuthenticationErrorResponseCie,
+    AuthenticationResponse, AuthenticationResponseCie)
+
+from .authn_responses_settings import (AUTHN_ERROR_RESPONSE_CIE,
```

Comment:
please execute `bash linting.sh` in the root of the project and then see `git diff`

Reply:
I did it but nothing has changed